Palace.xcodeproj/project.pbxproj

```diff
@@ -3815,6 +3815,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CURRENT_PROJECT_VERSION = 37;
 				CURRENT_PROJECT_VERSION = 38;
```

Comment:
Xcode created two version entries (again), could you re-type the build number to see if it's gone? I tried to re-type "38", it seems it removes the first line with "37"